pycm/pycm_obj.py

```diff
@@ -693,3 +694,25 @@ def weighted_average(self, param, weight=None, none_omit=False):
             return numpy.average(param_list, weights=weight_list)
         except Exception:
             return "None"
+
+    def weighted_kappa(self, weight=None):
+        """
+        Calculate weighted kappa.
+
+        :param weight: weight matrix
+        :type weight: dict
+        :return: weighted kappa as float
+        """
+        if matrix_check(weight) is False:
+            warn(WEIGHTED_KAPPA_WARNING, RuntimeWarning)
+            return self.Kappa
```

Comment:
Isn't it better to raise an error in the case of wrong weight parameter format?

Reply:
You are right, raising an error is a good option but I think for parameters like `weight` here and `alpha` in `CI` method that have a well-known default value, **warning** is a **better** option ;-)